Upstream-Status: Submitted
Signed-off-by: Ross Burton <ross.burton@intel.com>

From 0802aaaa9784813d318d045c79533a044eedf542 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Tue, 3 Mar 2015 11:32:29 +0000
Subject: [PATCH] tmpfiles.d: only copy /etc/pam.d if PAM is present

If HAVE_PAM isn't set then don't attempt to copy /etc/pam.d from the factory, as
it doesn't get installed.

Signed-off-by: Ross Burton <ross.burton@intel.com>
---
 tmpfiles.d/etc.conf.m4 |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/tmpfiles.d/etc.conf.m4 b/tmpfiles.d/etc.conf.m4
index 9b0e080..ab5cd16 100644
--- a/tmpfiles.d/etc.conf.m4
+++ b/tmpfiles.d/etc.conf.m4
@@ -14,4 +14,6 @@ m4_ifdef(`ENABLE_RESOLVED',
 L /etc/resolv.conf - - - - ../run/systemd/resolve/resolv.conf
 )m4_dnl
 C /etc/nsswitch.conf - - - -
+m4_ifdef(`HAVE_PAM',
 C /etc/pam.d - - - -
+)m4_dnl
-- 
1.7.10.4

